lib/img/local.js

```diff
@@ -43,6 +43,22 @@ LocalDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function localDelete(key) {
+    return new P(function deletePromise(resolve, reject) {
+      var dir = PUBLIC_DIR;
+      logger.debug('delete.start', key);
+      fs.unlink(path.join(dir, name), function(err) {
```

Comment:
What is `name` here? Is that supposed to be `key`? If so, are we relying on Hapi to do validation of `key` so it doesn't contain bad stuff like `../`?